charts/kured/README.md

```diff
@@ -56,6 +56,7 @@ The following changes have been made compared to the stable chart:
 | `configuration.slackChannel` | cli-parameter `--slack-channel`                                        | `""`                      |
 | `configuration.slackHookUrl` | cli-parameter `--slack-hook-url`                                       | `""`                      |
 | `configuration.slackUsername` | cli-parameter `--slack-username`                                      | `""`                      |
+| `configuration.notifyURL` | cli-parameter `--notify-url`                                              | `""`                      |
```

Comment:
Small nit: Can you rename the parameter from "notifyURL" to "notifyUrl" to be consistent with e.g. "slackHookUrl"?

Reply:
Sure. I thought I'd keep the name consistent with the one in the main.go code.